engine/Core.php

```diff
@@ -110,6 +114,9 @@ private function serve_child_module_asset($asset_path, $file_name) {
                         if (is_numeric($pos2)) {
                             $content_type = 'text/css';
                         }
+                        if (is_numeric($pos2)) {
```

Comment:
don't you need this line here `$pos2 = strpos($file_name, '.js');` ?

Reply:
Yeah - thought I'd added it. Cheers.

Reply:
Also, I had to add another check: if ($content_type == 'text/html') { $pos2 = strpos($file_name, '.js'); if (is_numeric($pos2)) { $content_type = 'text/javascript'; } }

When using string literals in the javascript it was returning html.